src/status_im/ui2/screens/chat/components/new_chat/view.cljs

```diff
@@ -69,18 +64,17 @@
      (let [contacts                          (rf/sub [:contacts/sorted-and-grouped-by-first-letter])
            selected-contacts-count           (rf/sub [:selected-contacts-count])
            selected-contacts                 (rf/sub [:group/selected-contacts])
-           window-height                     (rf/sub [:dimensions/window-height])
            one-contact-selected?             (= selected-contacts-count 1)
            contacts-selected?                (pos? selected-contacts-count)
            {:keys [primary-name public-key]} (when one-contact-selected?
                                                (rf/sub [:contacts/contact-by-identity
                                                         (first selected-contacts)]))
            no-contacts?                      (empty? contacts)]
-       [rn/view {:style {:height (* window-height 0.9)}}
+       [:<>
```

Comment:
The section-list in this file would need to be changed to use a `gesture/flat-list`, and have `scroll-enabled` and `on-scroll` parameters passed to it to be able to read gestures. Please open an issue if not for this PR.

Reply:
not moved because it must be refactored first